src/python.cc

```diff
@@ -642,12 +640,78 @@ TRITONBACKEND_ModelInstanceExecute(
   RETURN_IF_ERROR(TRITONBACKEND_ModelInstanceState(
       instance, reinterpret_cast<void**>(&instance_state)));
 
-  std::vector<TRITONBACKEND_Response*> responses;
-  responses.reserve(request_count);
-
+  ModelState* model_state =
+      reinterpret_cast<ModelState*>(instance_state->Model());
   uint64_t exec_start_ns = 0;
   SET_TIMESTAMP(exec_start_ns);
 
+  const int max_batch_size = model_state->MaxBatchSize();
```

Comment:
primitive datatypes do not need to be const.

Reply:
Fixed.